mkdocs_macros.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+from tabulate import tabulate
+
+# This file is for defining macros for mkdocs-macros plugin
+# Check https://mkdocs-macros-plugin.readthedocs.io/en/latest/macros/ for the details
+
+
+def format_param_type(param_type):
+    if param_type == "number":
+        return "float"
+    else:
+        return param_type
+
+
+def format_param_range(param):
+    list_of_range = []
+    if "enum" in param.keys():
+        list_of_range.append(param["enum"])
+    if "minimum" in param.keys():
+        list_of_range.append(">=" + str(param["minimum"]))
+    if "exclusiveMinimum" in param.keys():
+        list_of_range.append(">" + str(param["exclusiveMinimum"]))
+    if "maximum" in param.keys():
+        list_of_range.append("<=" + str(param["maximum"]))
+    if "exclusiveMaximum" in param.keys():
+        list_of_range.append("<" + str(param["exclusiveMaximum"]))
+    if "exclusive" in param.keys():
+        list_of_range.append("≠" + str(param["exclusive"]))
```

Comment:
If we can use "≠", then I think it would be nice to use "≥" and "≤" (or "⩾" and "⩽", but I think the first ones render better).